addon/mixins/ripple-mixin.js

```diff
@@ -1,287 +1,286 @@
 import Ember from 'ember';
-/* global Hammer */
-/* global propagating */
+const { inject, computed } = Ember;
+/* global window */
+
+const DURATION = 400;
 
 export default Ember.Mixin.create({
-  mousedown: true,
-  hover: true,
-  focus: true,
+  util: inject.service(),
+  rippleContainerSelector: '.md-container',
+
   center: false,
-  mousedownPauseTime: 150,
   dimBackground: false,
-  outline: false,
-  fullRipple: true,
-  isMenuItem: false,
   fitRipple: false,
+  colorElement: false,
+  noink: false,
 
-  isActive: false,
-  isHeld: false,
-  counter: 0,
+  rippleInk: computed('noink', 'rippleInkColor', function () {
+    if (this.get('noink')) {
+      return false;
+    }
+    if (this.get('rippleInkColor')) {
+      return this.get('rippleInkColor');
+    }
+    return "";
+  }),
 
-  ripples: [],
-  rippleStates: [],
+  didInsertElement() {
+    this.rippleElement = this.$(this.get('rippleContainerSelector'));
```

Comment:
let's call `this._super(...arguments);` to make sure we don't break inheritance.